spec/spec_helper.rb

```diff
@@ -143,6 +143,8 @@
   config.file_fixture_path = "spec/support/fixtures"
 end
 
+RSpec::Matchers.define_negated_matcher :not_change, :change
```

Comment:
The [RSpec docs](https://www.rubydoc.info/gems/rspec-expectations/3.5.0/RSpec%2FMatchers.define_negated_matcher) say this will lead to slightly confusing error messages:

>Note: While the most obvious negated form may be to add a not_ prefix, the failure messages you get with that form can be confusing (e.g. "expected [actual] to not [verb], but did not"). We've found it works best to find a more positive name for the negated form, such as avoid_changing rather than not_change.

I can live with that, but thought I'd mention it in case you wanted to change it.

Reply:
Good idea, I'll change it.